Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,249 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+4.What's the top 10 countries by SurfaceArea ?
+
+ SELECT Name, SurfaceArea FROM country ORDER BY SurfaceArea DESC LIMIT 10;
++--------------------+-------------+
+| Name               | SurfaceArea |
++--------------------+-------------+
+| Russian Federation | 17075400.00 |
+| Antarctica         | 13120000.00 |
+| Canada             |  9970610.00 |
+| China              |  9572900.00 |
+| United States      |  9363520.00 |
+| Brazil             |  8547403.00 |
+| Australia          |  7741220.00 |
+| India              |  3287263.00 |
+| Argentina          |  2780400.00 |
+| Kazakstan          |  2724900.00 |
++--------------------+-------------+
+10 rows in set (0.00 sec)
+
+5.What's the top 10 most populated cities?
+
+ SELECT Name, Population FROM country ORDER BY Population DESC LIMIT 10;
++--------------------+------------+
+| Name               | Population |
++--------------------+------------+
+| China              | 1277558000 |
+| India              | 1013662000 |
+| United States      |  278357000 |
+| Indonesia          |  212107000 |
+| Brazil             |  170115000 |
+| Pakistan           |  156483000 |
+| Russian Federation |  146934000 |
+| Bangladesh         |  129155000 |
+| Japan              |  126714000 |
+| Nigeria            |  111506000 |
++--------------------+------------+
+10 rows in set (0.00 sec)
+
+6.Select the top 3 countries by population that their names start with the letter ‘P’?
+
+ SELECT Name FROM country WHERE Name LIKE 'P%' ORDER BY Population DESC LIMIT 3;
++-------------+
+| Name        |
++-------------+
+| Pakistan    |
+| Philippines |
+| Poland      |
++-------------+
+3 rows in set (0.00 sec)
+
+7.What is the population (in) Asia? 
+
+ SELECT Population FROM country WHERE Continent = "Asia";
++------------+
+| Population |
++------------+
+|   22720000 |
+|    2441000 |
+|    3520000 |
+|    7734000 |
+|  129155000 |
+|     617000 |
+|     328000 |
+|    2124000 |
+| 1277558000 |
+|     754700 |
+|    4968000 |
+|    6782000 |
+|  212107000 |
+| 1013662000 |
+|   67702000 |
+|   23115000 |
+|    6217000 |
+|    5083000 |
+|  126714000 |
+|   16223000 |
+|    4699000 |
+|   11168000 |
+|   46844000 |
+|    1972000 |
+|    5433000 |
+|    3282000 |
+|   18827000 |
+|     473000 |
+|     286000 |
+|   45611000 |
+|    2662000 |
+|   22244000 |
+|   23930000 |
+|    2542000 |
+|  156483000 |
+|   75967000 |
+|   24039000 |
+|    3101000 |
+|     599000 |
+|   21607000 |
+|    3567000 |
+|   16125000 |
+|   61399000 |
+|    6188000 |
+|    4459000 |
+|     885000 |
+|   66591000 |
+|   22256000 |
+|   24318000 |
+|   79832000 |
+|   18112000 |
++------------+
+51 rows in set (0.00 sec)
+
+7.1 And what is the population of Asia ?
+
+ SELECT SUM(Population) FROM country WHERE Continent = "Asia" ;
```

Comment:
Yeah, this is what the question was really asking.

Reply:
Did you and kemirdin work together on this?

Reply:
no Mr.Jason i did not